src/util/settings/theme.cc

```diff
@@ -137,6 +137,10 @@ static QColor colorInvertedIfDark(QColor color) {
   return color;
 }
 
+QColor editedBackground() {
+  return colorInvertedIfDark(QColor("#AFAF00"));
```

Comment:
This is too similar to the selection color (at least in dark theme), and IMO too intensive.